lib/src/pages/actions/actions_model.dart

```diff
@@ -1,6 +1,61 @@
 import 'dart:io';
 
-void migrate() {
-  print("Migrating...");
-  Process.run("\$SNAP/bin/steam-snap.sh", []).then((value) => {print("Migrated library.")});
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:steam_tweaks/widgets.dart';
+import 'package:provider/provider.dart';
+import 'package:ubuntu_service/ubuntu_service.dart';
+import 'package:yaru_icons/yaru_icons.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+import 'actions_model.dart';
+
+void migrate(BuildContext context, String method) async {
+  final String steamGamesDirectory = "${Platform.environment['HOME']}/snap/steam/common/.local/share/Steam/steamapps/common";
+  if(method == "copy"){
+    print("Copied to snap!");
```

Comment:
Also, assuming the user has lots of games and the process takes a lot of time, it would be nice to add a progress bar. I'll investigate to see if this is possible.